<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>finalproj_re1032</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">A Collection of Slightly-Lower-than-Room-Temperature Rocks</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="rocks.html">Rocks</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="notes.html">Notes</a>
</li>
<li>
  <a href="finalproj_re1032.html">Final Project</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">finalproj_re1032</h1>

</div>


<p>Data analysis and Final Project for Data Science for Life Sciences (Fall2021).</p>
<p>The data analyzed here comes from an acyl-RAC experiment performed on <em>Arabidopsis thaliana</em>, a plant model organism. Acyl-RAC, or acyl-resin assisted capture, is a protein purification method that captures S-acylated proteins. S-acylation, or palmitoylation, is a form of post-translational modification where a 16 carbon fatty acid, palmitate, is attached to a surface exposed serine residue. Because palmitate is significantly hydrophobic, this pushes the protein to favor attachment to a lipid membrane, with the palmitate group inserted into the lipid layer.</p>
<div class="figure">
<img src="palmitoylation_noTMH_mechanism.png" alt="" />
<p class="caption">Palmitoylation can localize an otherwise cytosolic protein to a membrane</p>
</div>
<p>Here, the authors identified 5262 unique proteins.</p>
<p>Below is a table of predicted subcellular locations and the number of genes from the dataset that are predicted to occupy them. These locations are determined using SUBAcon, the SUBcellular Arabidopsis consensus algorithm. At the bottom, some proteins are predicted to have more than one location; though the significant majority of them occupy a single location in the cell. Nine of these proteins have no prediction noted.</p>
<table>
<caption>Table of Subcellular Protein locations as determined by SUBAcon</caption>
<thead>
<tr class="header">
<th align="left">Subcellular Location</th>
<th align="left">Proteins Counts</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">plastid</td>
<td align="left">804</td>
</tr>
<tr class="even">
<td align="left">nucleus</td>
<td align="left">854</td>
</tr>
<tr class="odd">
<td align="left">extracellular</td>
<td align="left">443</td>
</tr>
<tr class="even">
<td align="left">golgi</td>
<td align="left">257</td>
</tr>
<tr class="odd">
<td align="left">peroxisome</td>
<td align="left">123</td>
</tr>
<tr class="even">
<td align="left">cytosol</td>
<td align="left">1315</td>
</tr>
<tr class="odd">
<td align="left">nucleus,cytosol</td>
<td align="left">107</td>
</tr>
<tr class="even">
<td align="left">endoplasmic reticulum</td>
<td align="left">201</td>
</tr>
<tr class="odd">
<td align="left">mitochondrion</td>
<td align="left">386</td>
</tr>
<tr class="even">
<td align="left">plasma membrane</td>
<td align="left">528</td>
</tr>
<tr class="odd">
<td align="left">vacuole</td>
<td align="left">104</td>
</tr>
<tr class="even">
<td align="left">endoplasmic reticulum,golgi</td>
<td align="left">10</td>
</tr>
<tr class="odd">
<td align="left">mitochondrion,nucleus</td>
<td align="left">2</td>
</tr>
<tr class="even">
<td align="left">mitochondrion,plastid</td>
<td align="left">15</td>
</tr>
<tr class="odd">
<td align="left">vacuole,golgi</td>
<td align="left">6</td>
</tr>
<tr class="even">
<td align="left">plasma membrane,cytosol</td>
<td align="left">21</td>
</tr>
<tr class="odd">
<td align="left">endoplasmic reticulum,plastid</td>
<td align="left">1</td>
</tr>
<tr class="even">
<td align="left">extracellular,plasma membrane</td>
<td align="left">4</td>
</tr>
<tr class="odd">
<td align="left">golgi,nucleus</td>
<td align="left">1</td>
</tr>
<tr class="even">
<td align="left">endoplasmic reticulum,plasma membrane</td>
<td align="left">12</td>
</tr>
<tr class="odd">
<td align="left">endoplasmic reticulum,cytosol</td>
<td align="left">2</td>
</tr>
<tr class="even">
<td align="left">NA</td>
<td align="left">9</td>
</tr>
<tr class="odd">
<td align="left">golgi,plasma membrane</td>
<td align="left">9</td>
</tr>
<tr class="even">
<td align="left">golgi,cytosol</td>
<td align="left">8</td>
</tr>
<tr class="odd">
<td align="left">vacuole,plasma membrane</td>
<td align="left">7</td>
</tr>
<tr class="even">
<td align="left">mitochondrion,cytosol</td>
<td align="left">3</td>
</tr>
<tr class="odd">
<td align="left">nucleus,plasma membrane</td>
<td align="left">5</td>
</tr>
<tr class="even">
<td align="left">endoplasmic reticulum,extracellular</td>
<td align="left">4</td>
</tr>
<tr class="odd">
<td align="left">vacuole,cytosol</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">peroxisome,plastid</td>
<td align="left">2</td>
</tr>
<tr class="odd">
<td align="left">plastid,cytosol</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">extracellular,cytosol</td>
<td align="left">2</td>
</tr>
<tr class="odd">
<td align="left">mitochondrion,peroxisome</td>
<td align="left">1</td>
</tr>
<tr class="even">
<td align="left">extracellular,plastid</td>
<td align="left">1</td>
</tr>
<tr class="odd">
<td align="left">plasma membrane,golgi,cytosol</td>
<td align="left">2</td>
</tr>
<tr class="even">
<td align="left">nucleus,plastid,cytosol</td>
<td align="left">2</td>
</tr>
<tr class="odd">
<td align="left">vacuole,golgi,plasma membrane</td>
<td align="left">1</td>
</tr>
<tr class="even">
<td align="left">peroxisome,cytosol</td>
<td align="left">1</td>
</tr>
<tr class="odd">
<td align="left">mitochondrion,plasma membrane</td>
<td align="left">2</td>
</tr>
<tr class="even">
<td align="left">endoplasmic reticulum,vacuole</td>
<td align="left">1</td>
</tr>
</tbody>
</table>
<p>Next, proteins are organized according the MapMan bin they associated with. Similar to Gene Ontology terms, MapMan bins are a structured hierarchy of biological concepts originally developed specifically for <em>Arabidopsis</em>. Intensive studies have been done to compare the use of MapMan vs GO terms and found that it largely came down to a problem-specific condition.</p>
<p>The most interesting part is that of the 5262 proteins, 1678 of them are not assigned to any MapMan bin.</p>
<table>
<caption>Table of MapMan descriptors and Associated Protein Counts</caption>
<thead>
<tr class="header">
<th align="left">MapMan description</th>
<th align="left">Proteins counts</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Lipid metabolism</td>
<td align="left">173</td>
</tr>
<tr class="even">
<td align="left">not assigned</td>
<td align="left">1678</td>
</tr>
<tr class="odd">
<td align="left">Protein degradation</td>
<td align="left">211</td>
</tr>
<tr class="even">
<td align="left">Cell wall</td>
<td align="left">168</td>
</tr>
<tr class="odd">
<td align="left">Phytohormones</td>
<td align="left">117</td>
</tr>
<tr class="even">
<td align="left">Solute transport</td>
<td align="left">166</td>
</tr>
<tr class="odd">
<td align="left">Enzyme classification</td>
<td align="left">311</td>
</tr>
<tr class="even">
<td align="left">Vesicle trafficking</td>
<td align="left">181</td>
</tr>
<tr class="odd">
<td align="left">RNA processing</td>
<td align="left">146</td>
</tr>
<tr class="even">
<td align="left">Multi-process regulation</td>
<td align="left">24</td>
</tr>
<tr class="odd">
<td align="left">Cellular respiration</td>
<td align="left">109</td>
</tr>
<tr class="even">
<td align="left">Coenzyme metabolism</td>
<td align="left">118</td>
</tr>
<tr class="odd">
<td align="left">Protein translocation</td>
<td align="left">84</td>
</tr>
<tr class="even">
<td align="left">Amino acid metabolism</td>
<td align="left">131</td>
</tr>
<tr class="odd">
<td align="left">DNA damage response</td>
<td align="left">12</td>
</tr>
<tr class="even">
<td align="left">Carbohydrate metabolism</td>
<td align="left">129</td>
</tr>
<tr class="odd">
<td align="left">Secondary metabolism</td>
<td align="left">86</td>
</tr>
<tr class="even">
<td align="left">Protein biosynthesis</td>
<td align="left">309</td>
</tr>
<tr class="odd">
<td align="left">Redox homeostasis</td>
<td align="left">48</td>
</tr>
<tr class="even">
<td align="left">Cell cycle</td>
<td align="left">104</td>
</tr>
<tr class="odd">
<td align="left">Protein modification</td>
<td align="left">329</td>
</tr>
<tr class="even">
<td align="left">RNA biosynthesis</td>
<td align="left">151</td>
</tr>
<tr class="odd">
<td align="left">Nucleotide metabolism</td>
<td align="left">46</td>
</tr>
<tr class="even">
<td align="left">External stimuli response</td>
<td align="left">54</td>
</tr>
<tr class="odd">
<td align="left">Photosynthesis</td>
<td align="left">142</td>
</tr>
<tr class="even">
<td align="left">Cytoskeleton</td>
<td align="left">117</td>
</tr>
<tr class="odd">
<td align="left">Chromatin organisation</td>
<td align="left">84</td>
</tr>
<tr class="even">
<td align="left">Polyamine metabolism</td>
<td align="left">7</td>
</tr>
<tr class="odd">
<td align="left">Nutrient uptake</td>
<td align="left">27</td>
</tr>
</tbody>
</table>
<div class="figure">
<img src="finalproj_re1032_files/figure-html/unnamed-chunk-3-1.png" alt="Number of Predicted Transmembrane Helices of Acylated Proteins" width="672" />
<p class="caption">
Number of Predicted Transmembrane Helices of Acylated Proteins
</p>
</div>
<p>The above graph illustrates an important aspect of palmitoylation and membrane association. Transmembrane helices(TMH) are naturally occurring structures that favor insertion into lipid membranes and as such can become membrane associated without palmitoylation. The more TMH present in a protein, the strong the association to the point where some proteins do not assume their active conformation until they’ve inserted into a membrane. We see above that the overwhelming majority of proteins have no TMH, and would thus require palmitoylation in order to become membrane associated.</p>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
